web/wp-content/themes/dctx2/template-parts/content-blog.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-4kewkr.php, but found phpcs-scan-4KewkR.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).